contracts/src/v0.8/functions/dev/1_0_0/interfaces/IFunctionsRouter.sol

```diff
@@ -7,6 +7,13 @@ import {IRouterBase} from "./IRouterBase.sol";
  * @title Chainlink Functions Router interface.
  */
 interface IFunctionsRouter is IRouterBase {
+  /**
+   * @notice The identifier of the route to retrieve the address of the access control contract
+   * The access control contract controls which accounts can manage subscriptions
+   * @return id - bytes32 id that can be passed to the "getContractById" of the Router
+   */
+  function getAllowListId() external pure returns (bytes32);
```

Comment:
Not super elegant that we're using s_route for two different purposes but I think it's OK. I wonder if we could store the type alongside the route somehow...? Probably doesn't make sense.

Reply:
It was my original purposes that routes could be anything, so kinda as intended. That's why RouterBase.sol has the route keys as "id", not "donId".

That is interesting, something like,
```
  mapping(bytes32 => [address, string]) internal s_route; /* id => [contract address, label] */
```

I'm open to something like that, but unsure how useful it actually is. Since the `Routable` interface enforces `typeAndVersion` you could also just go `contract.typeAndVersion()` and save on storage writes in the Router.

Reply:
Might be just a waste of gas. Let's ignore for now.